flow/util/docker_shell

```diff
@@ -26,12 +29,15 @@ docker run -u $(id -u ${USER}):$(id -g ${USER}) \
  -e XAUTHORITY=$XAUTH \
  -e FLOW_HOME=/OpenROAD-flow-scripts/flow/ \
  -e MAKEFILES=/OpenROAD-flow-scripts/flow/Makefile \
- -v $WORKSPACE:`pwd` \
+ -e YOSYS_CMD=$YOSYS_CMD \
+ -e OPENROAD_EXE=$OPENROAD_EXE \
+ -e KLAYOUT_CMD=$KLAYOUT_CMD \
+ -v $WORKSPACE:/OpenROAD-flow-scripts/flow \
  $DOCKER_INTERACTIVE \
  ${OR_IMAGE:-openroad/flow-centos7-builder:latest} \
  bash -c "set -ex
  mkdir /tmp/xdg-run
- . ./env.sh
- cd $WORKSPACE
+ #. ./env.sh
```

Comment:
why commented out?

Reply:
otherwise looks good to me. I'm not using this, I just shared the docker_shell script....

Reply:
If you mount the docker image from the `flow` folder it seems that docker will not see any files above that directory. Hence `env.sh` will not appear in docker.